nautobot/extras/tests/test_jobs.py

```diff
@@ -604,3 +608,22 @@ def test_run(self):
         site_2 = Site.objects.filter(slug="test-site-two")
         self.assertEqual(site_2.count(), 1)
         self.assertEqual(site_2[0].cf["cf1"], "-")
+
+
+class RemoveScheduledJobManagementCommandTestCase(TestCase):
+    def test_remove_stale_scheduled_jobs(self):
+        for i in range(1, 7):
+            ScheduledJob.objects.create(
+                name=f"test{i}",
+                task="nautobot.extras.jobs.scheduled_job_handler",
+                job_class="local/test_pass/TestPass",
+                interval=JobExecutionType.TYPE_FUTURE,
+                user=self.user,
+                start_time=timezone.now() - datetime.timedelta(days=i * 30),
+                one_off=i % 2 == 0,  # True / False
+            )
+
```

Comment:
Might be good to include some recurring scheduled jobs that started some time ago but are still recurring, and ensure that they do *not* get deleted?